selectors-4/Overview.bs

```diff
@@ -2818,47 +2818,13 @@ The User-interaction Pseudo-classes: '':user-valid'' and '':user-invalid''</h4>
 	represent an element with incorrect or correct input, respectively,
 	but only <em>after</em> the user has significantly interacted with it.
 
-	The '':user-invalid'' pseudo-class
-	must match an '':invalid'', '':out-of-range'', or
-	blank-but-'':required'' elements between the time the user has attempted
-	to submit the form and before the user has interacted again with the
-	form element.
-
-	The '':user-valid'' pseudo-class must match a '':valid'' element between the
-	time the user has attempted to submit the form and before the user has
-	interacted again with the form element.
-
-	User-agents may allow them to match such elements at other times,
-	as would be appropriate for highlighting an error to the user.
-	For example, a UA may choose to have '':user-invalid'' match an
-	'':invalid'' element once the user has typed some text into it and
-	changed the focus to another element, and to stop matching only after
-	the user has successfully corrected the input.
-
-	<div class='example'>
```

Comment:
I'd leave this example in, it's a useful thing for the reader and matches what HTML defines.

Reply:
ok, i put the example back in